test/azure-authenticator.js

```diff
@@ -77,8 +78,8 @@ describe('azure-chaos', () => {
         resources: testObject.expectedResources,
         authKey: testObject.expectedAuthKey,
         accessToken: testObject.expectedAccessToken
-      }).then((res) => {
-        assert.throws(res.calledUri, null)
+      }).then(null, err => {
```

Comment:
nit: inconsistent styling - move the next `then` to the same line as closers, or move this `then` to it's own line